test/docker-compose-selenium-test.yml

```diff
@@ -6,8 +6,6 @@ services:
     build:
       context: ../Docker/test/selenium/
       dockerfile: Dockerfile
-    depends_on:
-      - wikibase
```

Comment:
Note: This was extraneous and perhaps not strictly correct, it makes it impossible to docker compose with this file for taking the test runner service down when the other services aren't running. It had no impact to remove it in terms of the test runs.